Update board contact info (#387)
datasektionen/bawang-content: en/organisation/sammansattning/body.md

# Members of the Board

The D-directorate is the board of the Computer Science chapter and consists of seven members.
They are the Chairman, Vice Chairman, Treasurer, secretary, and three more board members that are responsible
for educational issues, student welfare, and business relations. Together they arrange Chapter Meetings, make sure
the decisions at chapter meetings are executed, coordinate different parts of the chapter, and make strategic and long term plans.

Each board member is the contact person in the board for some of out elected volunteers a list can be found [here](https://dsekt.se/kontaktfunktionarare).

## The Chairman - Pontus Söderlund

The Chairman is in charge of the D-directorate and the contact with THS and other chapters.
The Chairman and the Vice Chairman work in close cooperation to create a plan and a vision with the rest of the board.

In addition to this The Chairman is an authorized signatory for the chapter and handles alcohol related questions.

> **Contact**

Reviewer: Is this your autoformatter going crazy?
I don't care really, just wondering about the diff.

The Chairman can be reached at [ordf@datasektionen.se](mailto:ordf@datasektionen.se).

## Vice chairman - Christofer Gärtner

The Vice chairmans primary role is to assist the Chairman and make sure
that the every board members workload is reasonable and that they are feeling well.
In the case that the Chairman can't fulfill his obligations the Vice chairman will
assume the role as acting Chairman.

The Vice chairman does many tasks that aren't related to any other board member or function in the chapter.

**Contact**

The Vice chairman can be reached at [vordf@datasektionen.se](mailto:vordf@datasektionen.se).

## The Treasurer - Felix Murnion

The Treasurer is responsible for the most things related to economics.
This includes declarations, bills, payments, and accounting. The Treasurer
is an authorized signatory for the chapter.

**Contact**

The Treasurer can be reached at [kassor@datasektionen.se](mailto:kassor@datasektionen.se).

## Vice treasurer

The Vice treasurer assists the Treasurer with most things related to economics.

**Contact**

The Vice treasurer can be reached at [vkassor@datasektionen.se](mailto:vkassor@datasektionen.se).

> ## The Secretary

Reviewer 2: ```suggestion
## The Secretary - *(Vacant)*
```
I think it's good to mark that a post is vacant explicitly.

Author: I think you're right, but I don't agree with it.

__Contact person to__

* Program responsible student

* The Chairman of the Study Council

* The Chairman of the Equality Committee

* International Student Coordinator
Works with business related question and our projects and committees related to this area.
Is also responsible for internal communications and that important information is available to our members.

* Student Safety Officer
**Contact**

* Project Leader for Project Pride
The Board member of Business Relations can be reached at [d-nok@datasektionen.se](mailto:d-nok@datasektionen.se).

## Board member of Student Issues - Malin Svenberg

Works with question regarding the well being of our members, working environment, and
social events.

__Contact__
**Contact**

The Board member of Student Issues can be reached at [d-ssf@datasektionen.se](mailto:d-ssf@datasektionen.se).

__Contact person to__

* Head of the Bar

* Chapter Sports Leader
## Board member of Educational Affairs - Abhinav Sasikumar

* Light and Sound responsible 

* Chief of Halls

* Cultural Attaché

* Prylis

* Mulle/Mullerina Schmeck

* DESCtop

> * Ärkedemon

Reviewer: If all of this is removed, have you made it explicit somewhere else?

Author: Yes, I added a link to an external document on line 8, [dsekt.se/kontaktfunktionarare](dsekt.se/kontaktfunktionarare).

Author: On second thought the link should probably be written from the perspective of the elected officials and not D-rek...

Author: Updated to [dsekt.se/kontaktperson](https://dsekt.se/kontaktperson).

> ## Board member of Study environment and Equality

Reviewer 2: ```suggestion
## Board member of Study environment and Equality - *(Vacant)*
```
See earlier suggestion

> Has overall responsibility for all issues connected to the physical and psychological study environment. Is the board's contact for questions relating to these areas. These responsibilities include

Reviewer 2: The translation of the beginning of this part feels a bit off.
```suggestion
Has an overall responsibility for all issues connected to the physical and psychological study environment. Is the person to contact on the board for questions relating to these areas. These responsibilities include
```

Author: You're right, in my defense it was very late when I wrote it :sweat_smile:

> - together with the Board member for Educational affairs coordinate the chapters communication concerning studies and the study environment towards EECS

Reviewer 2: ```suggestion
- coordinate the chapters communication concerning studies and the study environment towards EECS together with the Board member for Educational affairs
```
this order reads better in english
